docs/build/get-started.md

```diff
@@ -481,7 +481,7 @@ Again, depending on your setup, this may take a while as Fluence CLI may need to
 
 ```bash
 fluence service repl
-? Enter service name from fluence.yaml, path to a service or url to .tar.gz archive hello-world
+? Enter service name from fluence.yaml, path to a service or url to .tar.gz archive src/services/hello_world
```

Comment:
```suggestion
? Enter service name from fluence.yaml, path to a service or url to .tar.gz archive hello_world
```

Reply:
ah, okey, I checked, it works, sorry and thank you

Reply:
it works both with the service name and the path, I've checked :)

in future, we'll change it to a selection list, like with environments